.github/workflows/test-multiple-versions.yml

```diff
@@ -33,13 +33,13 @@ jobs:
           - 18.0.0
           - 18.1.0
           - 18.2.0
-          - 18.3.0-next-dd5365878-20230407
-          - 0.0.0-experimental-dd5365878-20230407
+          - 18.3.0-canary-aef7ce554-20230503
+          - 0.0.0-experimental-aef7ce554-20230503
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v2
         with:
-          node-version: '14'
+          node-version: ${{ startsWith(matrix.react, '16.') && '16' || '18' }}
```

Comment:
@dai-shi - This is because using v18 with `vi.useFakeTimers()` throws `performance.mark` doesn't exist` error.

Related: https://github.com/sinonjs/fake-timers/issues/420 , https://github.com/sinonjs/fake-timers/issues/427

Reply:
Wow, thanks!!